devices/philips.js

```diff
@@ -1863,7 +1863,7 @@ module.exports = [
             fz.legacy.hue_dimmer_switch, fz.battery],
         exposes: [e.battery(), e.action(['on-press', 'on-hold', 'on-hold-release', 'up-press', 'up-hold', 'up-hold-release',
             'down-press', 'down-hold', 'down-hold-release', 'off-press', 'off-hold', 'off-hold-release']),
-        exposes.numeric('action_duration', ea.STATE).withUnit('second')],
+        exposes.numeric('duration', ea.STATE).withUnit('second')],
```

Comment:
`duration` is legacy, it will show up once you disable the legacy integration: https://www.zigbee2mqtt.io/devices/324131092621.html#options

Reply:
Sorry, I remember. I close this.

Reply:
SORRY. I remember, we had this discussion in the past. I close this PR. THANKS !!!